lib/widgets/enable_backup_dialog.dart

```diff
@@ -283,7 +279,9 @@ class _BackupNowButtonState extends State<_BackupNowButton> {
               backupProvider: BackupProvider.remoteServer(),
               remoteServerAuthData: auth,
             ),
-          ).then((_) => Navigator.pop(context));
```

Comment:
popUntil wasn't used because backup prompt dialog can be shown on top of any route and we want to pop the dialog itself, not redirect users to home page.